### PR TITLE
Ensure that we get the event description (or post content) if it exists.

### PR DESCRIPTION
**Pull request by Camwyn**
Two things here:

1. The logic was inverted (so it was trying to get it if it was empty)
2. It's not always stored on the event as event->description, so if that's empty we try to use event->post_content

[TEC-4315]

[TEC-4315]: https://theeventscalendar.atlassian.net/browse/TEC-4315?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: readme.txt [hunk 1]
```diff
@@ -223,6 +223,10 @@ Remember to always make a backup of your database and files before updating!
 
 == Changelog ==
 
+= [TBD] TBD =
+
+* Fix - Ensure Google Calendar links include the event description if it exists. [TEC-4315]
```
Comment (jesseeproductions) [inline readme.txt:228]: Looks good, once the readme conflict is fixed it should be all set.

**Review by jesseeproductions — APPROVED** [review]

File: src/Tribe/Views/V2/iCalendar/Links/Google_Calendar.php [hunk 2]
```diff
@@ -115,7 +115,8 @@ public function generate_single_url( $post = null ) {
 			return '';
 		}
 
-		$event_details = empty( $event->description ) ? urlencode( $event->description ) : '';
+		$event_details = ! empty( $event->description ) ? $event->description : $event->post_content;
```
Comment (lucatume) [inline src/Tribe/Views/V2/iCalendar/Links/Google_Calendar.php:118]: Question: do we care that the event might have an empty `post_content`?

**Review by lucatume — APPROVED** [review]
Just one question.

File: src/Tribe/Views/V2/iCalendar/Links/Google_Calendar.php (hunk 2 above)
Comment (Camwyn) [reply src/Tribe/Views/V2/iCalendar/Links/Google_Calendar.php:118]: Good point, we should cover that possibility thoroughly. I'll revise.

**Review by jesseeproductions — APPROVED** [review]

**Pull request merged**